yarn.lock

```diff
@@ -4696,6 +4696,14 @@
     "@typescript-eslint/types" "5.10.2"
     "@typescript-eslint/visitor-keys" "5.10.2"
 
+"@typescript-eslint/scope-manager@5.9.1":
```

Comment:
these changes got added after I merged `master` into this branch. Not sure what happened there

Reply:
@udameli Can you merge/rebase again? This is now removed in https://github.com/getsentry/sentry/pull/31848

Reply:
thank you! fixed 🙏